src/tagsearch.rs

```diff
@@ -21,12 +21,13 @@ pub enum Flavor {
 }
 
 impl fmt::Display for Flavor {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        match self {
-            Flavor::Vim => f.write_str("Vim"),
-            Flavor::NeoVim => f.write_str("NeoVim"),
-            Flavor::Custom => f.write_str("custom"),
-        }
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
It's better to indicate anonymous lifetime in structs, even if it's not strictly necessary.